### PR TITLE
Add upload command to release-ci

### PR DESCRIPTION
**Pull request by csrwng**
`release-ci upload` will upload artifacts in the artifact directory

### CODE REVIEW

**Comment by csrwng** [comment]
@stevekuznetsov fyi ... I need to get this in so I can later do pipeline cleanup.

File: tools/release-ci/cmd/upload.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"errors"
+
+	"github.com/openshift/release/tools/release-ci/pkg/logging/gcs"
+)
+
+// sidecarCmd should run alongside a test pod and uploads files to GCS
```
Comment (stevekuznetsov) [inline tools/release-ci/cmd/upload.go]: godoc

File: tools/release-ci/cmd/upload.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"errors"
```
Comment (stevekuznetsov) [inline tools/release-ci/cmd/upload.go]: import ordering

**Comment by csrwng** [comment]
Fixing in followup

**Pull request merged**